studio/pages/new/[slug].tsx

```diff
@@ -436,39 +418,149 @@ const Wizard: NextPageWithLayout = () => {
+                              <Alert_Shadcn_ variant="warning">
+                                <IconAlertCircle className="h-4 w-4" />
+                                <AlertTitle_Shadcn_>
+                                  Your payment method will be charged
+                                </AlertTitle_Shadcn_>
+                                <AlertDescription_Shadcn_>
+                                  By creating a new Pro Project, there will be an immediate charge
+                                  of $25 once the project has been created.
+                                </AlertDescription_Shadcn_>
+                              </Alert_Shadcn_>
+                            )}
+                          </>
+                        }
+                      >
+                        {Object.entries(PRICING_TIER_LABELS).map(([k, v]) => {
+                          const label = `${v}${k === 'PRO' ? ' - $25/month' : ' - $0/month'}`
+                          return (
+                            <Listbox.Option key={k} label={label} value={k}>
+                              {label}
+                            </Listbox.Option>
+                          )
+                        })}
+                      </Listbox>
+                    )}
+
+                    {freePlanWithExceedingLimits && slug && (
+                      <div className={billedViaOrg ? '' : 'mt-4'}>
+                        <FreeProjectLimitWarning
+                          membersExceededLimit={membersExceededLimit || []}
+                          orgLevelBilling={billedViaOrg}
+                          orgSlug={slug}
+                        />
+                      </div>
+                    )}
 
-                <Panel.Content className="border-b border-panel-border-interior-light dark:border-panel-border-interior-dark">
-                  <Listbox
-                    layout="horizontal"
-                    label="Region"
-                    type="select"
-                    value={dbRegion}
-                    onChange={(value) => setDbRegion(value)}
-                    descriptionText="Select a region close to your users for the best performance."
-                  >
-                    {Object.keys(availableRegions).map((option: string, i) => {
-                      const label = Object.values(availableRegions)[i] as string
-                      return (
-                        <Listbox.Option
-                          key={option}
-                          label={label}
-                          value={label}
-                          addOnBefore={() => (
-                            <img
-                              alt="region icon"
-                              className="w-5 rounded-sm"
-                              src={`${router.basePath}/img/regions/${
-                                Object.keys(availableRegions)[i]
-                              }.svg`}
```

Comment:
Just making sure this doesn't get left out! We still need region selection ;)

Reply:
will fix

Reply:
ha, how did i accidentally leave this out 😂 
I was reordering stuff to make more sense when 'free plan' was selected